src/main/resources/templates/providerresponse/registration-service-languages.html

```diff
@@ -8,11 +8,11 @@
     <div class="grid">
       <div th:replace="~{fragments/goBack :: goBackLink}"></div>
       <main id="content" role="main" class="form-card spacing-above-35">
-        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header=#{placeholder}, subtext=#{placeholder})}"/>
+        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header='#{placeholder}', subtext='#{placeholder}')}"/>
```

Comment:
If you are running into issues with rendering this screen it's because placeholder is not a valid key. 
```suggestion
        <th:block th:replace="~{fragments/cardHeader :: cardHeader(header='placeholder', subtext='placeholder')}"/>
```

Reply:
I used the single quotes around `#{placeholder}`.  This converts it to a text literal so it shouldn't affect rendering.